lua/ggu.lua

```diff
@@ -9,6 +13,8 @@ return function (net_conf, gk_conf, lcore)
 	ggu_conf.lcore_id = lcore
 	ggu_conf.ggu_src_port = 0xA0A0
 	ggu_conf.ggu_dst_port = 0xB0B0
+	ggu_conf.mailbox_max_entries = mailbox_max_entries
```

Comment:
Replace `mailbox_max_entries` with `math.pow (2, mailbox_max_entries_exp) - 1`.

Reply:
So, for the ``ele_count`` parameter, we should use ``math.pow (2, mailbox_max_entries_exp)``, and for the ``rte_mempool_create()``, it should be ``ele_count - 1``?